### PR TITLE
fix: update tier values

### PR DESCRIPTION
**Pull request by cmunns**
Screen Shot 2022-10-12 at 12 17 28 PM

### CODE REVIEW

**Comment by heyjay44** [comment]
Can you coordinate merging with @drewdelano 's pricing page PR?

**Comment by gobengo** [comment]
This looks good to me. Someone told me not to merge it until we can coordinate with changes to the /pricing page too, which will come soon. I think we should reuse this same branch/PR for those changes too, which is the only reason I'm not marking this as approved in github yet. Thanks @cmunns!

**Comment by JeffLowe** [comment]
Blocked until Drew's updates to Pricing Page are ready to go on this branch/PR fyi @drewdelano 
( Pricing page update task in notion https://www.notion.so/pricing-page-change-tier-details-to-reflect-new-storage-prices-b3ad9e1d44784613ac6c9927cce36b5c

**Comment by JeffLowe** [comment]
@gobengo this is now unblocked and ready for review

File: packages/website/pages/pricing.js [hunk 1]
```diff
@@ -151,8 +151,8 @@ const PricingTiers = () => (
               title="Expert"
               price="$10"
               isBestValue={true}
-              storageAllocation="60GiB storage"
-              storageOverageRate="+ $0.17/mo per additional GiB"
+              storageAllocation="120GiB storage"
+              storageOverageRate="+ $0.075/mo per additional GiB"
```
Comment (gobengo) [inline packages/website/pages/pricing.js:155]: According to [reqs](https://www.notion.so/pricing-page-change-tier-details-to-reflect-new-storage-prices-b3ad9e1d44784613ac6c9927cce36b5c) it seems like this should be $.08/mo ?

Comment (gobengo) [inline packages/website/pages/pricing.js:155]:
```suggestion
              storageOverageRate="+ $0.08/mo per additional GiB"
```

**Review by gobengo — APPROVED** [review]

**Pull request merged**